src/views/menus/navigation-menu.js

```diff
@@ -64,9 +64,8 @@ export class NavigationMenu extends LitElement {
         <li>No files loaded</li>
       `;
     } else {
-      files.forEach(file => {
-        filesList = html`
-          ${filesList}
+      let filesArray = files.map(file => {
```

Comment:
minor: you can use the shorthand syntax here if you'd like:
```javascript
let filesArray = files.map(file => html`<li>...</li>`);
```

minor, naming: is there maybe a more specific name for this object instead of `array`?